index.d.ts

```diff
@@ -0,0 +1,69 @@
+import {PathLike} from 'line-column-path';
+
+declare namespace openEditor {
+	interface Options {
+		/**
+		Name, command, or binary path of the editor. Default: [Auto-detected](https://github.com/sindresorhus/env-editor).
```

Comment:
Let's put the `Default:` on it's own line.

Reply:
OK.